app/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,12 @@
     <string name="registerFailed">Registriation failed</string>
     <string name="cardImageDescription">Wizard card</string>
     <string name="gameActivity">Game Activity</string>
+    <!--LobbyActivity-->
+    <string name="LobbyCode">Code: %s</string>
```

Comment:
Currently all string resources start with a lowercase letter

Reply:
Changing this should be part of #33